apps/voting/test/voting.js

```diff
@@ -223,12 +223,12 @@ contract('Voting App', accounts => {
                     assert.isTrue(isOpen, 'vote should be open')
                     assert.isFalse(isExecuted, 'vote should not be executed')
                     assert.equal(creator, holder51, 'creator should be correct')
-                    assert.equal(snapshotBlock, await getBlockNumber() - 1, 'snapshot block should be correct')
-                    assert.equal(supportRequired.toNumber(), neededSupport.toNumber(), 'required support should be app required support')
-                    assert.equal(minQuorum.toNumber(), minimumAcceptanceQuorum.toNumber(), 'min quorum should be app min quorum')
+                    assert.equal(snapshotBlock.toString(), await getBlockNumber() - 1, 'snapshot block should be correct')
+                    assert.equal(supportRequired.toString(), neededSupport.toString(), 'required support should be app required support')
+                    assert.equal(minQuorum.toString(), minimumAcceptanceQuorum.toString(), 'min quorum should be app min quorum')
```

Comment:
Another alternative could be to use `BN#eq`: 
```js
assert(minQuorum.eq(minimumAcceptanceQuorum), 'min quorum should be app min quorum')
```

Reply:
It's actually using `Bignumber.js` since we're still on truffle@4, but I do like using the string comparison since it means less migrations in the future :)